### DISCUSSION TITLE
All map participants should be mentioned and displayed on the user profile.

### DISCUSSION THREAD

**Discussion by ghost** [ppy/osu#27415] [Ideas]
Nowadays, making a Beatmap is no longer a single person job. The participants in a map are multifaceted, such as hitsound, storyboard, gds, but now only gd mentioned, and gd can only be associated as one. If a beatmap is created by more than three people working together, then the existing data is unfair to some people. I believe that all users involved in creating a map should be mentioned and displayed on the user profile, Currently, only a portion of the gd is displayed, and many users who produce hitsound and storyboard have not received more display. I think this workload will be quite heavy and challenging, but I still think it is necessary to do this in order to ensure that all map participants receive fair treatment.

**Comment by Joehuu** [comment]
Relevant web issue/proposal: https://github.com/ppy/osu-web/issues/6908

**Reply by ghost** [reply]
4 years ago... But the BG issue mentioned in this issue may not be easy to implement, but GDer hitsounder storyboarder can be quickly implemented,just need a lot of time to supplement the database. I don't understand why we didn't work together at the beginning, which led to the need to start over with previous supplementary work.
